src/cljs/main/broadfcui/page/workspace/summary/catalog/wizard.cljs

```diff
@@ -60,23 +60,27 @@
 
 (def ^:private ALL_USERS "All users")
 
+(defn- make-list [input]
+  ;;input may or maynot be a list, make it a list
+  (flatten (conj '() input)))
+
 (react/defc DiscoverabilityPage
   {:validate (constantly nil)
    :get-initial-state
    (fn [{:keys [props]}]
-     (select-keys props [:library:discoverableByGroups :owner?]))
+     {:library:discoverableByGroups (make-list (get props :library:discoverableByGroups '()))})
```

Comment:
if `ensure-sequence` (née `make-list`) is nil-safe, you don't need the default value here.